conda-store-server/tests/user_journeys/test_user_journeys.py

```diff
@@ -29,15 +29,32 @@ def test_admin_user_can_create_environment(
     base_url: str, token: str, specification_path: str
 ) -> None:
     """Test that an admin user can create an environment."""
-    namespace = utils.API.gen_random_namespace()
     api = utils.API(base_url=base_url, token=token)
-    api.create_namespace(namespace)
-    response = api.create_environment(namespace, specification_path)
-    data = response.json()["data"]
-    assert "build_id" in data
-    build_id = data["build_id"]
-    assert build_id is not None
-    build = api.wait_for_successful_build(build_id)
-    environment_name = build.json()["data"]["specification"]["name"]
-    api.delete_environment(namespace, environment_name)
+    namespace = "default"
+    environment = api.create_environment(namespace, specification_path).json()["data"][
+        "specification"
+    ]["name"]
+    api.delete_environment(namespace, environment)
+
+
+@pytest.mark.user_journey
+@pytest.mark.parametrize(
+    "specification_path",
+    [
+        ("tests/user_journeys/test_data/simple_environment.yaml"),
+    ],
+)
+def test_login_and_delete_shared_environment(
+    base_url: str, specification_path: str
+) -> None:
+    """Test that a user can login and delete an environment in a shared namespace."""
```

Comment:
This is specifically testing an admin user correct? would it be worth adding another pytest parameter for the token and creating a non admin token to test this for both?